.circleci/config.yml

```diff
@@ -577,6 +577,36 @@ jobs:
       - store_test_results:
           path: target/reports/
 
+  itest-lambda-event-source-mapping-v2-feature:
```

Comment:
Would be great to also run the MA/MR tests on these changes to make sure that these tests are multi-region and multi-accounts compatible, otherwise this change could break the scheduled configured pipeline run.
https://github.com/localstack/localstack/blob/master/docs/testing/multi-account-region-testing/README.md

Reply:
Re-triggered because pushing a new commit canceled the manually triggered run: https://app.circleci.com/pipelines/github/localstack/localstack/27180/workflows/ac64fa2a-e673-4391-83f8-14e97f907323

That's probably not intended (just a minor inconvenience).

Reply:
all 🟢